packages/devtools_server/lib/src/server.dart

```diff
@@ -341,6 +347,7 @@ Future<void> registerLaunchDevToolsService(
           // to the containers loopback IP).
           path: devToolsUri.path.isEmpty ? '/' : devToolsUri.path,
           queryParameters: uriParams,
+          fragment: page,
         );
 
```

Comment:
it would be nice to have an integration test that exercises this functionality. The integration test is actually serving devtools using the server so it should be possible to do today.

Reply:
Yeah, that's what my comment was above above. As it turns out, to get the tests working on CI we need to run Chrome with the debug port open (see https://github.com/flutter/devtools/pull/1120#issuecomment-537904248) so that might give me a good route to see what page is selected (though we could equally track it on the server and include it in the response to `client.list`).